sound/soc/intel/boards/sof_da7219.c

```diff
@@ -545,6 +545,10 @@ static const struct platform_device_id board_ids[] = {
 		.name = "adl_mx98360_da7219",
 		/* no quirk needed for this board */
 	},
+	{
+		.name = "rpl_mx98360_da7219",
+		/* no quirk needed for this board */
+	},
```

Comment:
can you clarify the commit message, specifically "MCLK frequency must be 12.288 or
24.576MHz." Isn't this set in the topology and not under the control of the kernel?

Reply:
Yes, it should be set in topology. The commit message just explains why we use 12.288/23.576 MHz MCLK instead of 19.2 MHz.

Reply:
then let's edit the commit message to say "DA7219 uses PLL bypass mode to avoid WCLK locking problem. To use this mode, the MCLK frequency must be 12.288 or 24.576MHz in the topology binary."